Add work and about section to my branch.

norbiboy1 opened the pull request:
In the portfolio mobile version setup I added the next:

- created the work section(card section) with the images, buttons;
- created the about myself section with the languages in the ul list;
- stylized in the css file all of the above sections with Grid and Flexbox.

Commit:
Add work and about section to my branch.

prolajumokeoni commented:
hi @norbiboy1 kindly add a descriptive pull request message 👍🏿

norbiboy1 reviewed (commented):
In the portfolio mobile version setup I added the next:
 - created the work section(card section) with the images, buttons;
 - created the about myself section with the languages in the ul list;
 - stylized in the css file all of the above sections with Grid and Flexbox.

prolajumokeoni commented on portfolio.html:
```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
     </section>
 
+    <section class="work">
+        <h1 class="recent-works">My Recent Works</h1>
```
- [ ] Kindly make the header tag any hx tag from h2 - h6,  this will be more semantic for your website  👍🏿

prolajumokeoni commented on portfolio.html:
```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
+                <img src="./images/mobile_version/Card/Img_Plaholder.png" class="image" alt="image placeholder2">
+                
+                <div class="content">
+                <h1 class="header-work-1">Multi-Post Stories <br> Gain+Glory</h1>
```
- [ ] Kindly make the header tag any hx tag from h2 - h6,  this will be more semantic for your website  👍🏿

prolajumokeoni commented on portfolio.html:
```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
+                <img src="./images/mobile_version/Card/Img_Plaholder.png" class="image" alt="image placeholder3">
+                
+                <div class="content">
+                <h1 class="header-work-1">Multi-Post Stories <br> Gain+Glory</h1>
```
- [ ] Kindly make the header tag any hx tag from h2 - h6,  this will be more semantic for your website  👍🏿

prolajumokeoni commented on portfolio.html:
```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
+                <img src="./images/mobile_version/Card/Img_Plaholder.png" class="image" alt="image placeholder4">
+                
+                <div class="content">
+                <h1 class="header-work-1">Multi-Post Stories <br> Gain+Glory</h1>
```
- [ ] Kindly make the header tag any hx tag from h2 - h6,  this will be more semantic for your website  👍🏿

prolajumokeoni commented on portfolio.html:
```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
+                <img src="./images/mobile_version/Card/Img_Plaholder.png" class="image" alt="image placeholder6">
+                
+                <div class="content">
+                <h1 class="header-work-1">Multi-Post Stories <br> Gain+Glory</h1>
```
- [ ] Kindly make the header tag any hx tag from h2 - h6,  this will be more semantic for your website  👍🏿

prolajumokeoni commented on portfolio.html:
```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
+                <img src="./images/mobile_version/Card/Img_Plaholder.png" class="image" alt="image placeholder7">
+                
+                <div class="content">
+                <h1 class="header-work-1">Multi-Post Stories <br> Gain+Glory</h1>
```
- [ ] Kindly make the header tag any hx tag from h2 - h6,  this will be more semantic for your website  👍🏿

prolajumokeoni commented on portfolio.html:
```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
 
     </section>
 
+    <section class="work">
```
- [ ] Kindly add the appropriate button  styling for  all buttons in this section, please see the design, and follow the button styling for Active, Hover, and Pressed
- [ ] Please remove the border outline of all buttons, this will make your buttons look exactly like the Figma design 👍🏿 
![Screenshot_395](https://user-images.githubusercontent.com/69638013/185760017-d8da5417-e31e-4b7d-bb3d-f5bfcf36e283.png)

ichala replied:
- [ ] like the previous reviewers requested , kindly add the styling for Active, Hover, and Pressed , for each button on your page 

![image](https://user-images.githubusercontent.com/89282221/185765590-fd0c00d3-29c0-4ce5-8e57-eceed020cfa5.png)

prolajumokeoni commented on portfolio.html:
```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
+
+    <section class="about-me">
+        <div class="text-about-me">
+            <h1 class="about">About me</h1>
```
- [ ] Kindly make the header tag any hx tag from h2 - h6,  this will be more semantic for your website  👍🏿

prolajumokeoni commented on portfolio.html:
```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
+            <img src="./images/mobile_version/Card/illustration-1-about-me-mobile.png" class="about-me-mobile" alt="illustration icon">
+            <p class="about-me-text"> I can help you build a product , <br> feature or website Look through <br> some of my work and experience! If <br> you like what you see and have a <br> project you need coded, <br>
+                don’t hestiate to contact me.</p>
+            <button type="button" class="get-my-resume">Get My Resume</button>
```
- [ ] Kindly add the appropriate button  styling for  all buttons in this section, please see the design, and follow the button styling for Active, Hover, and Pressed
- [ ] Please remove the border outline of all buttons, this will make your buttons look exactly like the Figma design 👍🏿 
![Screenshot_395](https://user-images.githubusercontent.com/69638013/185760017-d8da5417-e31e-4b7d-bb3d-f5bfcf36e283.png)

prolajumokeoni commented on portfolio.html:
```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
+        <h1 class="recent-works">My Recent Works</h1>
+        <img src="./images/mobile_version/Indicator/Indicator.png" class="indicator_horizontal" alt="indicator icon">
+
+        <div class="main-content-cards">
```
- [ ] Kindly avoid using static widths in your design currently, all your cards are aligned to the left and they should be at the center please fix this
- [ ] please fix the first li; ruby on rails for each card, all li's should have the same width 👍🏿 


https://user-images.githubusercontent.com/69638013/185760961-5c8c82f8-eebb-40be-8d4f-5d411997345c.mp4

prolajumokeoni reviewed (changes requested):
Hi @norbiboy1, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Change the button properties on hover;
Change the language lists size;
Change the headings to h2 - h6;
Fix the position of the main-content-card.

Commit:
Linter configurations check

Kamba56 commented on portfolio.css:
```diff
@@ -211,3 +207,731 @@ body {
+  margin: 0;
+}
+
+.btn-green:hover {
+  border: 1px solid blue;
+  text-decoration: underline;
+  text-decoration-color: blue;
+}
+
```
- [ ] As mentioned by the previous reviewer. Please make sure that your button hover and pressed state matches the one in the figma design template. You can use the `:active` pseudo-class for the pressed state
![Screenshot 2022-08-20 203951](https://user-images.githubusercontent.com/94551484/185763627-ce6a6ba5-cc72-401b-8d1c-2d803bbcbd54.png)
.

Kamba56 commented on portfolio.css:
```diff
@@ -211,3 +207,731 @@ body {
+}
+
+.ruby {
+  width: 120px;
+  padding: 8px 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  background: #ebf0ee;
+  color: #3a4a42;
+  border-radius: 4px;
+  list-style: none;
+}
+
+.css {
+  width: 45px;
+  padding: 8px 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  background: #ebf0ee;
+  color: #3a4a42;
```
- [Optional] It is best practice not to repeat yourself when writing code. The properties in the `.ruby   .css   .HTML    .javacript` are identical. You should consider using just a single class to represent them. This will make your code smaller and better. You can read more about DRY code [here](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

Kamba56 commented on portfolio.css:
```diff
@@ -211,3 +207,731 @@ body {
+  margin: 0;
+}
+
+.ruby {
+  width: 120px;
+  padding: 8px 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  background: #ebf0ee;
+  color: #3a4a42;
+  border-radius: 4px;
+  list-style: none;
+}
+
+.css {
+  width: 45px;
+  padding: 8px 12px;
```
- [ ] As requested by the [previous code reviewer, please remove all widths for the technologies list. It is best practice for responsive web design not to use static widths. You can remove the widths and use padding to create spacing in the box.

- [ ] As requested by the previous code reviewer, kindly adjust the contents of your work and about section. The contents seem to be overflowing to the right. Kindly fix that, please.

Figma template*
![Screenshot 2022-08-20 204757](https://user-images.githubusercontent.com/94551484/185763882-8ace61fb-abd2-4022-b77e-619f8d04c459.png)

*your work*
![Screenshot 2022-08-20 204724](https://user-images.githubusercontent.com/94551484/185763884-c0452b5d-6bbf-4740-8d30-1f5e4ee333aa.png)

Kamba56 reviewed (changes requested):
Hi @norbiboy1, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] Linters checks passed
- [x] Descriptive PR title and description
- [x] Correct use of GitHub flow


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Fix the position;
Remove width from header, nav, and sections;
Remove width from the languages, frameworks and skills

ichala commented on portfolio.html:
```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
+        <h2 class="recent-works">My Recent Works</h2>
+        <img src="./images/mobile_version/Indicator/Indicator.png" class="indicator_horizontal" alt="indicator icon">
+
+        <div class="main-content-cards">
```
- [ ] you still didn't fix the overflow in your website , kindly fix it as the previous reviewers requested 

![REVOVR](https://user-images.githubusercontent.com/89282221/185765664-afb32672-c311-4417-afd1-a6b28d845416.gif)

ichala reviewed (changes requested):
Hi @norbiboy1 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

![the-simpsons-homer-simpson](https://user-images.githubusercontent.com/28307960/157369993-50d5ba12-099f-4f2a-9a19-5e57ee2d1c96.gif)

## To Highlight 🎉

- No Linters Errors ✔️
- Descriptive PR title and description ✔️


 ## Required Changes ♻️

- [ ]  _**_Check The Comments Below_**_

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Overflow issue fix

HAMMAS-SALEEM commented on portfolio.css:
```diff
@@ -403,6 +404,10 @@ body {
+.btn-green:enabled {
+  background-color: #36b37f;
+}
+
```
### Effects
![effects](https://user-images.githubusercontent.com/89282221/185765590-fd0c00d3-29c0-4ce5-8e57-eceed020cfa5.png)
- [ ] Your button effects are not working. Please use :hover instead of :enabled :blush:
- [ ] You didn't added these effects properly. :thinking:
**Please add these effects as previous reviewer asked :+1:**

HAMMAS-SALEEM reviewed (changes requested):
# Required Changes :recycle:
![almost-there](https://user-images.githubusercontent.com/93039359/178569876-9fcda5e8-6819-4569-95ee-acb393903a05.jpg)

### Hey @norbiboy1 :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Meaningful commits
:heavy_check_mark: Added .gitignore
:heavy_check_mark: Following GitHub Flow

Good job on implementing some of the changes requested by the previous reviewer, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional Suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. Otherwise reach out to me on slack which is more fast. You can reach out to me on slack by searching my name in **full-time-students** channel :+1:

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
I use the hover, neither enabled to show the button pressing function.

codecaiine reviewed (approved):
Hi @norbiboy1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: No linter errors
:heavy_check_mark: Git flow was correctly used
:heavy_check_mark: Good design
:heavy_check_mark: Good PR title and Description


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
